launch_ros/launch_ros/substitutions/find_package.py

```diff
@@ -28,7 +30,7 @@
 from launch.utilities import perform_substitutions
 
 
-@expose_substitution('find-pkg')
+@expose_substitution('find-pkg-prefix')
 class FindPackage(Substitution):
```

Comment:
@jacobperron nit: should this be renamed to `FindPackagePrefix`?

Reply:
See my comment above: https://github.com/ros2/launch_ros/pull/57#discussion_r321421929